ios/FluentUI.Demo/FluentUI.Demo/Demos/SideBarDemoController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class SideBarDemoController: DemoController {
+	private var sideBar: SideBar?
+	private var contentViewController: UIViewController?
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		container.addArrangedSubview(createButton(title: "Show side bar", action: #selector(presentSideBar)))
+	}
```

Comment:
Should use loadView instead. loadView is used when constructing a hierarchy programmatically rather than using a xib, viewDidLoad is used for additional configuration after loading from a xib. Documentation pending at https://microsoft.github.io/apple-ux-guide/ViewControllers.html#construct-view-hierarchies-in-loadview